hs_core/search_indexes.py

```diff
@@ -141,28 +145,30 @@ def get_content_types(res):
 class BaseResourceIndex(indexes.SearchIndex, indexes.Indexable):
     """Define base class for resource indexes."""
 
-    text = indexes.CharField(document=True, use_template=True)
+    text = indexes.CharField(document=True, use_template=True, stored=False)
```

Comment:
so everything you turned stored=False, those fields are not being used for display, correct?

Reply:
Stored = false means several things: 
1. That the field is not explicitly stored in SOLR. 
2. That SOLR responses do not contain the field (increasing speed). 
3. That getting the field requires hitting the Django database, which SOLR does automatically.